src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double loadFactor = 0.75;
```

Comment:
how about naming? it should be LOAD_FACTOR
the same about initialCapacity, let's make it constant